srtcore/queue.h

```diff
@@ -549,6 +549,7 @@ struct CMultiplexer
 #ifdef SRT_ENABLE_IPOPTS
    int m_iIpTTL;
    int m_iIpToS;
+   std::string m_BindToDevice;
```

Comment:
`#ifdef SRT_ENABLE_BINDTODEVICE` is missing.
Is it used?

Reply:
Fixed.